content/docs/setup/behavioral_changes.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Behavioral Changes in 1.1
+description: Major changes in behavior, both backward compatibility breaking and non, in Istio 1.1.
+weight: 15
+icon: setup
+
+---
+
+# Major Behavioral Changes in Istio 1.1
+
+## Installer Changes
```

Comment:
The page needs an overall intro paragraph before diving into the main content.

"This page describes..."

Reply:
We generally don't talk about "Installer", so I'd just go with "Installation" instead.

Reply:
done